src/backend/src/prisma/schema.prisma

```diff
@@ -290,3 +306,21 @@ model Risk {
   deletedByUserId  Int?
   deletedBy        User?     @relation(name: "deletedBy", fields: [deletedByUserId], references: [userId])
 }
+
+model Task {
+  taskId          String        @id @default(uuid())
+  title           String
+  notes           String
+  deadline        DateTime
+  assignees       User[]        @relation(name: "assignedTo")
+  priority        Task_Priority
+  status          Task_Status
```

Comment:
Same here, might want a default

Reply:
^

Reply:
I could see status having one, but I feel like priority shouldn't bc/ like peyton said it should be passed in by the user when we create one, and I don't think there's an easy default like "in backlog" probably is for the status. Imo tho neither of these need need a default